buildkite/resource_team_member_test.go

```diff
@@ -22,9 +21,9 @@ func TestAccTeamMember_add_remove(t *testing.T) {
 				Config: testAccTeamMemberConfigBasic("MEMBER"),
 				Check: resource.ComposeAggregateTestCheckFunc(
 					// Confirm the team member exists in the buildkite API
-					testAccCheckTeamMemberExists("buildkite_team_member.test", &resourceTeamMember),
+					testAccCheckTeamMemberExists("buildkite_team_member.test", &tm),
 					// Confirm the team member has the correct values in Buildkite's system
-					testAccCheckTeamMemberRemoteValues(&resourceTeamMember, "MEMBER"),
+					testAccCheckTeamMemberRemoteValues(&tm, "MEMBER"),
```

Comment:
Knit, but passing in `string` then `&reference` in the first function, but then `&reference` then `string` in the second is a little jarring.

Reply:
Solid - flipping the `testAccCheckTeamMemberRemoteValues` params in the adjustment commit